src/components/Form/ActionButton/Prompt.js

```diff
@@ -135,25 +101,23 @@ ${mailFooter}`);
               className="light inline-icon-button toggle-comment-button"
               onClick={switchToFullEditMode}
             >
-              <EditIcon color="var(--grey)" size={17} />
+              Editer <EditIcon color="var(--grey)" size={17} />
```

Comment:
can you align the icon with the text ? 
![image](https://user-images.githubusercontent.com/4190728/97892279-1b626100-1d30-11eb-8d3f-270c975224c6.png)


there are some classes that can be useful : https://github.com/betagouv/signup-front/blob/master/src/components/Form/ActionButton/index.js#L196-L209